frame/balances/src/lib.rs

```diff
@@ -369,6 +369,26 @@ pub mod pallet {
 			<Self as Currency<_>>::transfer(&transactor, &dest, value, KeepAlive)?;
 			Ok(().into())
 		}
+
+		/// Transfer the entire transferable balance from the caller account.
+		///
+		/// # <weight>
+		/// - O(1). Just like transfer, but reading the user's transferable balance first.
+		/// #</weight>
+		#[pallet::weight(T::WeightInfo::transfer_all())]
+		pub fn transfer_all(
+			origin: OriginFor<T>,
+			dest: <T::Lookup as StaticLookup>::Source,
+			keep_alive: bool,
+		) -> DispatchResultWithPostInfo {
+			use fungible::Inspect;
```

Comment:
Why is this needed?

Reply:
It was complaining I could not call `::reducible_balance()` without this import.